Header header
bool test_started
bool test_finished
# External Status from Solver
int32 solver_status
float32 cost_value
float32 kkt_value
float32 sol_time